packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -38,16 +24,7 @@ export class SupportComponent {
 	 */
 	constructor( page: Page ) {
 		this.page = page;
-	}
-
-	/**
-	 *
-	 */
-	async waitForQueryComplete(): Promise< void > {
-		await Promise.all( [
-			this.page.waitForSelector( selectors.resultsPlaceholderHelpCenter, { state: 'hidden' } ),
-			this.page.waitForSelector( selectors.resultsPlaceholder, { state: 'hidden' } ),
-		] );
+		this.root = this.page.getByLabel( 'Help Center' );
```

Comment:
This is a technique/style I learned from working on the Gutenberg Playwright migration, specifically [this comment](https://github.com/WordPress/gutenberg/pull/45784#discussion_r1189538014).

The idea is that, when using role-based selectors (as recommended by Playwright now), we trade legibility for some ambiguity in the locator structure. For instance, the following locator is easy to read and understand, but there is potential to match on multiple elements on a page.

```
page.getByRole('button', { name: 'home' });
```

By always selecting from a parent/root locator it is possible to understand the overall structure hierarchy.

Note: if I was following the Gutenberg code style, this would probably be named something like `this.helpCenter`instead of `this.root`.